cypress/e2e/requests/filter-request-by-tags.cy.js

```diff
@@ -0,0 +1,74 @@
+import testType from '../../support/dictionary/testTypes';
+import TopMenu from '../../support/fragments/topMenu';
+import Requests from '../../support/fragments/requests/requests';
+import Users from '../../support/fragments/users/users';
+import DevTeams from '../../support/dictionary/devTeams';
+import InventoryInstance from '../../support/fragments/inventory/inventoryInstance';
+import { ITEM_STATUS_NAMES, REQUEST_TYPES } from '../../support/constants';
+
+describe('ui-requests: Filter requests by tags', () => {
+  const userIds = [];
+  const requests = [];
+  const instances = [];
+  const requestTypes = { PAGE: 'Page', HOLD: 'Hold', RECALL: 'Recall' };
+  let oldRulesText;
+  let requestPolicyId;
```

Comment:
Please remove unused variables

Reply:
Steps for updating Circulation Rules and Delete Request Policy were added in afterHook so  variables from 14 and 15 rows are necessary now.